tq42/exception_handling.py

```diff
@@ -21,18 +21,24 @@ def wrapped(*args, **kwargs):
         except InactiveRpcError as e:
             status_code = e.code()
             if status_code == StatusCode.PERMISSION_DENIED:
-                raise exceptions.PermissionDeniedError() from None
+                print(exceptions.PermissionDeniedError())
+                return list()
 
             if status_code == StatusCode.UNAUTHENTICATED:
-                raise exceptions.UnauthenticatedError() from None
+                print(exceptions.UnauthenticatedError())
```

Comment:
as below. Either we have an error or not but we are not doing it like this.... 🙈